plugins/chat/spec/support/chat_service_matcher.rb

```diff
@@ -71,12 +71,7 @@ def matches?(service)
       end
 
       def has_error?
-        result[step].errors.present?
```

Comment:
I removed this because it would error in the scenario I have with `LookupThread`, where this contract step would not exist if the initial `threaded_discussions_enabled` policy does not pass, so my test for "when params are not valid" would just fail with an error.

At any rate I don't see why this `failure_message` is needed anyway, the `FailStep` one fits better IMO

Reply:
Indeed, that’s better with the default `failure_message`. I think those are just leftovers and we can actually refactor these matchers a bit now that we normalized everything regarding the result keys.

I’m going to open a small refactor PR :+1:

Reply:
Cool sounds good 👍